kubernetes/src/main/java/io/kubernetes/client/custom/BaseExponent.java

```diff
@@ -0,0 +1,56 @@
+package io.kubernetes.client.custom;
+
+public class BaseExponent {
```

Comment:
JavaDoc?

Reply:
Happy to add JavaDoc - to clarify, is this an across the board ask or just for `BaseExponent`?